src/reducer/prodFilterReducer.js

```diff
@@ -0,0 +1,93 @@
+const HIGH_TO_LOW = "HIGH_TO_LOW";
```

Comment:
export this constant from file action>type.js so that it can be reused in other components also.